cg/meta/create_validation_cases/validation_case_data.py

```diff
@@ -0,0 +1,121 @@
+"""Model for down sampling meta data."""
+
+import logging
+from pathlib import Path
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants import DataDelivery, Priority
+from cg.constants.constants import CaseActions, CustomerId
+from cg.meta.create_validation_cases.validation_data_input import ValidationDataInput
+from cg.store.models import ApplicationVersion, Case, Sample, Customer
+from cg.store.store import Store
+from cg.utils.calculations import multiply_by_million
+
+LOG = logging.getLogger(__name__)
+
+
+class ValidationCaseData:
+    def __init__(self, status_db: Store, validation_data_input: ValidationDataInput):
+        """Initialize the validation sample data and perform integrity checks."""
+        self.status_db: Store = status_db
```

Comment:
The class name sounds a bit like it is a pure data object, but it contains database access logic. Maybe there is a better way to encapsulate the logic/data? Not sure

Reply:
it contains data only but uses access logic to make sure the stored data is correct. It is similar to the downsampledata class.